plugins/velociraptor_legacy/icon_velociraptor_legacy/connection/connection.py

```diff
@@ -0,0 +1,98 @@
+import json
+import paramiko
+import base64
+import grpc
+import io
+import time
+from pyvelociraptor import api_pb2
+from pyvelociraptor import api_pb2_grpc
+import insightconnect_plugin_runtime
+from .schema import ConnectionSchema, Input
+
+# Custom imports below
+
+
+class Connection(insightconnect_plugin_runtime.Connection):
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+
+    def connect(self):
+        # START INPUT BINDING - DO NOT REMOVE - ANY INPUTS BELOW WILL UPDATE WITH YOUR PLUGIN SPEC AFTER REGENERATION
+        # TODO: generate bound input variables for the user, to help handhold the user
+        # TODO: ex. self.api_key = params.get(Input.API_KEY)
+        # END INPUT BINDING - DO NOT REMOVE
+        self.logger.info("Connect: Connecting...")
+        """
+        Runs a VQL query against the Velociraptor server.
+
+        Args:
+            config: A dictionary containing the configuration parameters for the Velociraptor server.
+            query: The VQL query to run.
+
+        Returns:
+            A tuple containing the query, the response, and the query execution logs.
```

Comment:
I don't think this docstring is relevant here, it is also causing one of the checks to fail. Kindly remove it 😃